networking/hub-regionA.bicep

```diff
@@ -651,8 +651,10 @@ resource fwPolicy 'Microsoft.Network/firewallPolicies@2021-05-01' = {
                 '${split(environment().authentication.loginEndpoint, '/')[2]}' // Prevent the linter from getting upset at login.microsoftonline.com
                 '*.blob.${environment().suffixes.storage}' // required for the extension installer to download the helm chart install flux. This storage account is not predictable, but does look like eusreplstore196 for example.
                 'azurearcfork8s.azurecr.io' // required for a few of the images installed by the extension.
-                '*.docker.io' // Only required if you use the default bootstrapping manifests included in this repo. Kured is sourced from here by default.
-                '*.docker.com' // Only required if you use the default bootstrapping manifests included in this repo. Kured is sourced from here by default.
+                '*.docker.io' // Only required if you use the default bootstrapping manifests included in this repo.
+                '*.docker.com' // Only required if you use the default bootstrapping manifests included in this repo.
+                'ghcr.io' // Only required if you use the default bootstrapping manifests included in this repo. Kured is sourced from here by default.
+                'pkg-containers.githubusercontent.com' // Only required if you use the default bootstrapping manifests included in this repo. Kured is sourced from here by default.
```

Comment:
not so sure about the last one in here.

Reply:
that's required because that's where the actual images from when the pull happens:  ghcr.io ends up doing a redirect at pull time to this location.

<img width="805" alt="image" src="https://user-images.githubusercontent.com/671075/201951296-cbe00fa3-9f91-4fc6-8265-b4a7aa04d623.png">

Reply:
got it, thanks for sharing the info.